awx/api/views/webhooks.py

```diff
@@ -83,7 +83,9 @@ def get_event_status_api(self):
 
     def get_event_ref(self):
         key = self.ref_keys.get(self.get_event_type(), '')
+        print("key:", key)
         value = self.request.data
+        print("value:", value)
```

Comment:
Spare debugging here. (I haven't looked at the rest of the PR, just noticed this.)